ssl/quic/quic_wire_pkt.c

```diff
@@ -39,12 +40,16 @@ int ossl_quic_hdr_protector_init(QUIC_HDR_PROTECTOR *hpr,
 
     hpr->cipher = EVP_CIPHER_fetch(libctx, cipher_name, propq);
     if (hpr->cipher == NULL
-        || quic_hp_key_len != (size_t)EVP_CIPHER_get_key_length(hpr->cipher))
+        || quic_hp_key_len != (size_t)EVP_CIPHER_get_key_length(hpr->cipher)) {
+        ERR_raise(ERR_LIB_SSL, ERR_R_EVP_LIB);
```

Comment:
Again there are cases here where we raise an err, but in other codepaths above we do not.